fvm/src/call_manager/backtrace.rs

```diff
@@ -77,8 +76,6 @@ pub struct Frame {
     pub source: ActorID,
     /// The method that was invoked.
     pub method: MethodNum,
-    /// The parameters passed to this method.
-    pub params: RawBytes,
```

Comment:
This could have gone in a separate PR, but it makes the send/invoke refactor patch a bit simpler (no need to clone the params).